src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between ;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+.header__logo {
+  margin-left: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
```

Comment:
Your header center aligns items vertically. There is no need for logo to have equal top and bottom margins. Try to delete them and see the result